.github/workflows/ci.yml

```diff
@@ -1,9 +1,13 @@
-# .github/workflows/ci.yml
 name: ci
 
-on: [push, pull_request]
+env:
+  PLUGIN: tool_cloudmetrics
```

Comment:
Still not sure why this has changed, either this is right and the docs are wrong and should be fixed or vice versa

https://github.com/catalyst/catalyst-moodle-workflows#add-the-workflow

Reply:
https://wiki.wgtn.cat-it.co.nz/wiki/Catalyst_Australia_Pty_Ltd/Elearning/best_practices/publishing-a-plugin#2._Set_.60PLUGIN.60_environment_variable

Reply:
Hmm I think the one in github is cannonical and more correct @keevan can you check this? I think I'll delete all the stuff in the wiki

Reply:
Yeah we should be using the one here now:
https://github.com/catalyst/catalyst-moodle-workflows

Reply:
@srdjan-catalyst I have cleaned up the wiki, we should not need the env: PLUGIN either this can be inferred

Reply:
But we DO need the secret as per the github docs

Reply:
Hi @srdjan-catalyst ,

You should be able to get the release workflow working by replacing everything here with the default example in the add the workflow section, with some minimal tweaks:
https://github.com/catalyst/catalyst-moodle-workflows#add-the-workflow

- You'll need to uncomment the "secret:" and "moodle_org_token" lines so they are no longer comments)
- You'll need to get rid of the `with:` statement, if there's no options this plugin needs to pass through.